Xpotify/Controls/NowPlayingView.xaml

```diff
@@ -493,6 +493,7 @@
                                                                         IsThumbToolTipEnabled="True"
                                                                         Value2="{x:Bind ViewModel.Volume, Mode=TwoWay}"
                                                                         Maximum="100"
+                                                                        PointerWheelChanged="SliderExtended_PointerWheelChanged"
```

Comment:
Please rename this event to `VolumeSlider_PointerWheelChanged` to match the other event of this element.